utils/build-toolchain-tensorflow

```diff
@@ -131,20 +143,28 @@ elif [ "$(uname -s)" == "Linux" ]; then
 fi
 
 
-# Report the commands being run
+# Report the commands being run.
 set -x
 YEAR=$(date +"%Y")
 MONTH=$(date +"%m")
 DAY=$(date +"%d")
 TOOLCHAIN_VERSION_DATE="5.0.${YEAR}${MONTH}${DAY}"
-TOOLCHAIN_VERSION="swift-tensorflow-LOCAL-${YEAR}-${MONTH}-${DAY}-a"
-ARCHIVE="${TOOLCHAIN_VERSION}-${HOST}.tar.gz"
-SYM_ARCHIVE="${TOOLCHAIN_VERSION}-osx-symbols.tar.gz"
+
+# Set toolchain name based on whether this is a normal or release toolchain.
+if [[ -z ${S4TF_RELEASE_VERSION} ]]; then
+  TOOLCHAIN_NAME="swift-tensorflow-DEVELOPMENT-${YEAR}-${MONTH}-${DAY}-a"
+  DISPLAY_NAME_SHORT="Swift for TensorFlow Development Snapshot"
+  DISPLAY_NAME="${DISPLAY_NAME_SHORT} ${YEAR}-${MONTH}-${DAY}"
+else
+  TOOLCHAIN_NAME="swift-tensorflow-RELEASE-${S4TF_RELEASE_VERSION}"
+  DISPLAY_NAME_SHORT="Swift for TensorFlow ${S4TF_RELEASE_VERSION} Release"
+  DISPLAY_NAME="${DISPLAY_NAME_SHORT}"
+fi
+
+ARCHIVE="${TOOLCHAIN_NAME}-${HOST}.tar.gz"
+SYM_ARCHIVE="${TOOLCHAIN_NAME}-osx-symbols.tar.gz"
 BUNDLE_PREFIX=com.google.swift
 BUNDLE_IDENTIFIER="${BUNDLE_PREFIX}.${YEAR}${MONTH}${DAY}"
-DISPLAY_NAME_SHORT="Swift for TensorFlow Local Snapshot"
```

Comment:
A minor functional change is that the default Darwin toolchain display name is now `Swift for TensorFlow Development Snapshot` instead of `Swift for TensorFlow Local Snapshot`.

This change is intentional. The new display names are consistent with S4TF [development snapshot](https://github.com/tensorflow/swift/blob/master/Installation.md#development-snapshots) and [release](https://github.com/tensorflow/swift/blob/master/Installation.md#releases) macOS toolchains, which lets use directly use `utils/build-toolchain-tensorflow` for building those toolchains instead of internal modified versions.